doc/source/installation.rst

```diff
@@ -65,6 +65,13 @@ available on your controller or host machines.
   highly recommended to upgrade user setuptools even when using a proper
   virtualenv as shown above.
 
+Keep in mind that on selinux supporting systems, if you install into a virtual
+environment, you may face https://github.com/ansible/ansible/issues/34340 even
```

Comment:
Probably better to: 1) make this a link with RST syntax 2) explain the issue briefly.

Reply:
[1] FWIW it should be auto-linked anyway. So not a big deal to me.